data/FilterListTag.json

```diff
@@ -7762,5 +7762,117 @@
   {
     "filterListId": 1640,
     "tagId": 10
+  },
+  {
+    "filterListId": 1673,
+    "tagId": 110
```

Comment:
It looks like you are linking these new lists to a tag that doesn't exist [here](https://github.com/collinbarrett/FilterLists/blob/master/data/Tag.json).

Reply:
I´ve confused language settings and the category. (And actually, I prefer that I don’t have a language setting attached to it, as it covers every domain)
I will modify it with the Anti-corp one (19).

Reply:
Ah, ok, no worries. Yeah, you can have any number of tags attached to any list. They are the small colored tags in the UI on the website.

Also, all the info we collect is documented over in the [Wiki](https://github.com/collinbarrett/FilterLists/wiki).

Reply:
I’ve hesitated to also add the Privacy tag, but as it doesn’t only point those specifically, i’ve decided to not use it and stick with the Anti-corp one.